python/ray/dashboard/client/src/api.ts

```diff
@@ -0,0 +1,95 @@
+const base =
+  process.env.NODE_ENV === "development"
+    ? "http://localhost:8080"
+    : window.location.origin;
+
+// TODO(mitchellstern): Add JSON schema validation for the responses.
+const get = async <T>(path: string, params: { [key: string]: any }) => {
+  const url = new URL(path, base);
+  for (const [key, value] of Object.entries(params)) {
+    url.searchParams.set(key, value);
+  }
+
+  const response = await fetch(url.toString());
+  const json = await response.json();
+
+  const { result, error } = json;
+
+  if (error !== null) {
+    throw Error(error);
+  }
+
+  return result as T;
+};
+
+export interface NodeInfoResponse {
```

Comment:
Please add comments to the field that's not self-explainartory. Because we are not using json-schema right now. brief comments to the following fields and their unit will be appreciated:
- boot_time
- cpu
- cpus
- mem
- net
- memory_full_info

Reply:
Done.